template.yaml

```diff
@@ -1,67 +1,95 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  Hello World lambda
+  A lambda for sending monthly cloud-spend emails for STRIDES accounts
 
-Metadata:
-  AWS::ServerlessRepo::Application:
-    Name: "lambda-template"
-    Description: "A GH template for quickly starting a new AWS lambda project."
-    Author: "Sage-Bionetworks"
-    SpdxLicenseId: "Apache-2.0"
-    # paths are relative to .aws-sam/build directory
-    LicenseUrl: "LICENSE"
-    ReadmeUrl: "README.md"
-    Labels: ["serverless", "template", "github", "quick-start"]
-    HomePageUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template"
-    SemanticVersion: "0.0.3"
-    SourceCodeUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template/tree/0.0.3"
+Parameters:
+  ScheduleExpression:
+    Type: String
+    Description: Schedule expressions using cron
```

Comment:
couldn't this also be a `rate expression`?  https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-scheduled-rule-pattern.html

Reply:
I'm not sure if that can be passed to the lambda as an environment variable.